docs/kyma/04-07-local-installation-deprecated.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Install Kyma locally
+type: Installation
+---
+
+>**WARNING:** Deprecated TBD
```

Comment:
Please write something so that technical write how to pimp it.

Installation based on scripts are deprecated, please use installation based on Kyma CLI, see XXX. The installation based on scripts will be removed with Kyma 1.4